patches/all/exit-hook+1.1.1.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/node_modules/exit-hook/index.js b/node_modules/exit-hook/index.js
+index e18013f..3366356 100644
+--- a/node_modules/exit-hook/index.js
++++ b/node_modules/exit-hook/index.js
+@@ -14,9 +14,9 @@ function exit(exit, signal) {
+ 		el();
+ 	});
+ 
+-	if (exit === true) {
+-		process.exit(128 + signal);
+-	}
++	// if (exit === true) {
++	// 	process.exit(128 + signal);
++	// }
```

Comment:
This patch removes the need for calling `process.removeAllListeners('SIGINT');`, which was deleted in this PR. The obvious problem with calling `process.removeAllListeners('SIGINT');` is that it does not allow multiple SIGINT handlers to be installed at the same time (including by CLI dependencies), and selectively removed. It was a poor design decision by the author of `exit-hook` to call `process.exit` -- it assumes that `exit-hook` is the only dependency that installs signal handlers, that the application will never need more than one handler for SIGINT, and that the application does not require graceful exit with centralised error handling (CLI's `lib/errors.ts`). We cannot get rid of `exit-hook` just yet because it is used by some dependencies. Hence patching.